components/product/ProductInfo.tsx

```diff
@@ -189,7 +192,7 @@ function ProductInfo({ page, layout }: Props) {
               <summary class="cursor-pointer">Descrição</summary>
               <div
                 class="ml-2 mt-2"
-                dangerouslySetInnerHTML={{ __html: description }}
+                dangerouslySetInnerHTML={{ __html: descriptionHtml ?? description }}
```

Comment:
This will change the behaviour for products that currently have the `descriptionHtml`, would like to check with @tlgimenes and @igorbrasileiro if this is a good default behaviour for them

Reply:
yeah, this is catalog dependent. Storefront should be independent of the catalog

Reply:
It makes sense to me that, for the shopify integration,  the HTML description be automatically available on storefront. Another solution would have been to have put the HTML description in the regular description attribute of the product, but that would mean having to choose between one of them and maybe putting an on and off switch for this behaviour at the shopify app configuration file (where we add in the api key).
If not, how else should this be approached ? Should we keep the descriptionHtml in the product's additionalProperty attribute and make use of it in my own personal site, not changing the storefront repo ?